simul/platform/cliutils.go

```diff
@@ -98,11 +100,9 @@ func Build(path, out, goarch, goos string, buildArgs ...string) (string, error)
 	buildBuffer := bufio.NewWriter(&b)
 	wd, _ := os.Getwd()
 	log.Lvl4("In directory", wd)
-	var args []string
-	args = append(args, "build", "-v")
-	args = append(args, buildArgs...)
-	args = append(args, "-o", out, path)
-	cmd = exec.Command("go", args...)
+	// we have to cd into the directory to do the build when using go
+	// modules, not sure about the exact reason for this behaviour yet
+	cmd = exec.Command("/bin/bash", "-c", fmt.Sprintf("(cd %s && go build -v %s -o %s)", path, strings.Join(buildArgs, " "), out))
```

Comment:
I have seen students launch simulations from Windows, and this will fail because it depends on /bin/bash. Instead, please use https://godoc.org/os/exec#Cmd.Dir instead to set the working directory.